saleor/static/dashboard-next/pages/components/PageDetailsPage/PageDetailsPage.tsx

```diff
@@ -112,9 +112,9 @@ const PageDetailsPage: React.StatelessComponent<PageDetailsPageProps> = ({
                 onChange={change}
               />
               <CardSpacer />
-              <PageAvailability
+              <VisibilityCard
                 data={data}
-                disabled={disabled}
+                loading={disabled}
```

Comment:
...but why?
It would make sense if you'd show skeleton instead of target component, but all you achieved with this change is this prop flow: `disabled => loading => disabled`